readersampleprovider.go

```diff
@@ -202,18 +200,19 @@ func (p *ReaderSampleProvider) NextSample() (media.Sample, error) {
 		sample.Duration = time.Duration(p.ivfTimebase*float64(delta)*1000) * time.Millisecond
 		p.lastTimestamp = header.Timestamp
 	case webrtc.MimeTypeOpus:
-		pageData, pageHeader, err := p.oggreader.ParseNextPage()
+		// TODO(theomonnom): Read next OpusPacket instead of the next page
+		data, _, err := p.oggreader.ParseNextPage()
```

Comment:
does ParseOpusPacketDuration rely on the fact that oggreader returns a packet, instead of a page?

Reply:
This is already working if the page contains only one opus packet.
But this isn't the right thing to do, we should correctly read the pages & segments inside the ogg container.